frontend/src/exams/generate-financial-report-modal.vue

```diff
@@ -56,8 +56,8 @@
             endDate: '',
             options: [
               {text: 'ITA - Individual and Group ', value: 'ita'},
-              {text: 'Pesticide', value: 'pesticide'},
-              {text: 'Bulk Milk Tank Grader', value: 'milk_tank'}
+              {text: 'Veterinary Exam', value: 'veterinary'},
```

Comment:
We will need to be able to search for pesticide exams, I think. Do you think there should be an "other" as well, that filters everything not in the above?

Reply:
pushed up